runtime/libia2/ia2.c

```diff
@@ -436,6 +444,10 @@ int protect_pages(struct dl_phdr_info *info, size_t size, void *data) {
     search_args->found_library_count++;
   }
 
+  if (syslib) {
+    printf("IA2: Protecting system library %s in compartment 1\n", libname);
```

Comment:
You can use `ia2_log` which works just like printf but automatically prefixes the function name